p2p/opts.go

```diff
@@ -115,3 +116,9 @@ func NewDHT(ctx context.Context, storageDir string, host host.Host) (*dht.IpfsDH
 	return dht.New(ctx, host, dhtopts.Datastore(store), dhtopts.Protocols(dhtProtocolID))
 
 }
+
+// NewDHTWithDatastore returns a new Kademlia DHT instance configured with store
+// as the persistant storage interface.
+func NewDHTWithDatastore(ctx context.Context, store datastore.Batching, host host.Host) (*dht.IpfsDHT, error) {
```

Comment:
I don't think we necessarily need to put this here. The main point of __p2p/opts.go__ and __p2p/opts_js.go__ is to allow for different options when running Mesh in the browser and running in standalone mode. Since it looks like this is only used for bootstrap nodes we could move it to __cmd/mesh-bootstrap__.

Reply:
moved to **cmd/mesh-bootstrap**